### ISSUE TITLE
DM synthetic images on ResNet-18

### ISSUE THREAD

**Issue by Yearsflow** [VICO-UoE/DatasetCondensation#30]
Hi,

Thanks for the work. I follow the experimental setting and generate 1 condensed image per class for MNIST with ResNet-18 using Group Normalization with DSA augmentation. However, the test accuracy is about 10% ~ 20%. I've read the other issues about ResNet-18. I wonder if it is normal, or if I can continue to improve the test accuracy.

Here are my hyperparameter settings.

`method=DM, dataset=MNIST, batch_real=64, dis_metric='ours', dsa_strategy='color_crop_cutout_scale_rotate', init='real', ipc=1, iter=20000, lr_img=1.0, batch_train=64, lr_net=0.01`

Furthermore I want to run experiments on medical image classification datasets, such as ISIC-2020, EyePACS. Would you please give some advice on how to set `iteration` and `lr_img` hyperparameters? Should I keep using ResNet-18 as backbone?

I would appreciate it if you answer my questions.

**Comment by BAAI-DCAI** [comment]
Hi,

You can try smaller learning rate: 0.1, 0.01 for lr_img. You'd better to use ConvNet as the base model to learn synthetic data. It can achieve good performance when training new architectures with the synthetic data.

**Comment by Yearsflow** [comment]
Thanks for your advice. It means that if I insist on using ResNet-18 as the base model I cannot achieve better test accuracy, right? I have to replace ResNet-18 with ConvNet as the base model.

**Comment by BAAI-DCAI** [comment]
Yes. It is better to use ConvNet. Training with ResNet18 is tricky. ResNet18 needs carefully tuning.

**Comment by Yearsflow** [comment]
Thank you very much! It helps a lot.
